pyproject.toml

```diff
@@ -48,6 +48,7 @@ docs = [
 ]
 test = [
   "filelock >= 3",
+  'importlib-metadata; python_version<"3.8"',
```

Comment:
This shouldn't be necessary, we have `importlib-metadata` on line 38.

Reply:
Oops, I saw it in typing and didn't look to see if it was in dependencies! Thanks!